restful_api/myapi/api/schemas/official_enroll.py

```diff
@@ -0,0 +1,16 @@
+from myapi.models import OfficialEnroll
+from myapi.extensions import ma, db
+
+
+class OfficialEnrollSchema(ma.SQLAlchemyAutoSchema):
+    
+    eng_id = ma.Int()
+    course_id = ma.Int()
+    start_date = ma.Date()
+    end_date = ma.Date()
+    has_passed = ma.Boolean()
```

Comment:
I think these lines are unnecessary if you declare `OfficialEnrol` under `Meta`

Reply:
it made a difference during testing. Without these lines, the endpoint returned `start_date` , `end_date` and `has_passed` only. Hm we KIV, I try fix in the future? Lacking full understanding of marshmallow now

Reply:
Oh okay maybe my understanding off. Leave it as it is bah